OpenBullet/Pages/Main/Configs/ConfigManager.xaml.cs

```diff
@@ -148,9 +149,16 @@ public void SaveConfig()
             Current.Config.Settings.LastModified = DateTime.Now;
             Current.Config.Settings.Version = Globals.obVersion;
             Globals.LogInfo(Components.ConfigManager, "Converted the unbinded observables and set the Last Modified date");
-            
+
+            if (File.Exists(Current.Path) && ((Current.Config.Settings.Name + ".loli") != Current.Path.Replace(Globals.configFolder + "\\", "")))
+            {
+                string newpath = Globals.configFolder + "\\" + (Current.Category == "Default" ? "" : (Current.Category + "\\")) + Current.Config.Settings.Name + ".loli";
+                File.Move(Current.Path, newpath);
```

Comment:
What if the new config name has invalid windows characters like < or : or > ?
You didn't take account for those. I made a static method in the BlockBase class called `MakeValidFileName` you NEED to call it before doing this if you don't want the program to crash upon bad characters!